cmd/login-consent-server/templates/consent.html

```diff
@@ -175,8 +172,41 @@ <h1 class="my-4 text-5xl font-bold leading-tight"></h1>
     }
 
 </script>
+<script type="text/javascript">
+    // todo : Add friendly name for the scope issue-508
+    $(document).ready(function () {
+            if(document.getElementById("scopeName").innerText == "mDL"){
```

Comment:
This would be better done by creating an object that maps from scope to friendly name, and then looking up.

E.g., https://ultimatecourses.com/blog/deprecating-the-switch-statement-for-object-literals

Reply:
done